### PR TITLE
Portguide v2 edits

### PR DESCRIPTION
**Pull request by abkro**
Edited device configuration page + corrected two links that had changed.

### CODE REVIEW

File: porting/UBports_installer.rst [hunk 1]
```diff
@@ -3,4 +3,6 @@ Configuring the UBports installer
 
 *Work in progress*
 
-For the time being, refer to the UBports porting notes on `the UBPorts installer and UBports recovery <https://github.com/ubports/porting-notes/wiki/HowTo:-UBports-Installer-&-System-Image>`_.
\ No newline at end of file
+For the time being, Halium 7.1 porters should refer to the UBports porting notes on `the UBPorts installer and System image <https://github.com/ubports/porting-notes/wiki/HowTo:-UBports-Installer-&-System-Image-(Halium-7.1)>`_.
+
+There is currently no guide that covers these steps for Halium 9 porting.
```
Comment (doniks) [inline porting/UBports_installer.rst:8]: I'm not too convinced of the  "currently no guide" statement. By definition the docs are not the authoritative source for what is and isn't documented in the pads and wikis. Assume that tomorrow someone writes a guide for this in the wiki. Now the docs are actually hindering the porting process by telling porters that there is nothing, even though there is.

File: porting/UBports_recovery.rst [hunk 2]
```diff
@@ -3,4 +3,6 @@ Building UBports recovery
 
 *Work in progress*
 
-For the time being, refer to the UBports porting notes on `the UBPorts installer and UBports recovery <https://github.com/ubports/porting-notes/wiki/HowTo:-UBports-Installer-&-System-Image>`_.
\ No newline at end of file
+For the time being, Halium 7.1 porters should refer to the UBports porting notes on `the UBPorts installer and System image <https://github.com/ubports/porting-notes/wiki/HowTo:-UBports-Installer-&-System-Image-(Halium-7.1)>`_.
+
+There is currently no guide that covers these steps for Halium 9 porting.
```
Comment (doniks) [inline porting/UBports_recovery.rst:8]: same

**Review by doniks — CHANGES_REQUESTED** [review]

File: porting/UBports_installer.rst (hunk 1 above)
Comment (abkro) [reply porting/UBports_installer.rst]: OK. Removed that line.

File: porting/UBports_recovery.rst (hunk 2 above)
Comment (abkro) [reply porting/UBports_recovery.rst]: Same as above.

**Review by doniks — APPROVED** [review]

**Pull request merged**